### ISSUE TITLE
pr4100 install error

### ISSUE THREAD

**Issue by sneakypete73** [WDCommunity/wdpksrc#22]
i installed COPS via the .bin as a manual app install in the web ui of my drive and it seemed to install. it says its running, but when i click configure i get configuration check page - screenshot attached. the app wont uninstall now either - it simply hangs at updating and when i relaunch the web ui it still shows as installed. am i missing something?
![2018-10-14](https://user-images.githubusercontent.com/39318326/46922787-1f669d80-d006-11e8-8865-333b5e3ca86a.png)

**Comment by stefaang** [comment]
Yeah the COPS package is a confusing one without the full description.
See the full guide on https://community.wd.com/t/install-calibre-cops-on-pr2100/219966

Basically, the error here says that /shares/Public/metadata.db is not found, which means you didn't configure Calibre on your PC to store its database on your Public share.
The COPS package uses an existing Calibre database and provides a web front-end for it.

**Comment by sneakypete73** [comment]
Thanks Stefaan.

 

To be clear. Do I install Calibre on my PC – as opposed to my NAS?

Once installed and pointing to my directory COPS will then work?

Does the PC need to be constantly running for COPS on the NAS to work?

**Comment by sneakypete73** [comment]
I am sorry to bother you my friend. I have changed the path as you instructed but I am still getting an error – can you please check the attached and confirm.

 

Thanks

 

Mike

**Comment by stefaang** [comment]
Please have a look at the description of [COPS](https://github.com/seblucas/cops): 
It says Calibre is nice software but you don't want to run Calibre on a NAS for its content server.

Install the [official Calibre](https://calibre-ebook.com/download) and point your ebook library to a directory on a share on your My Cloud NAS, e.g. the Public share.
Any ebooks you add via Calibre will end up in your Calibre Library, which resides on the NAS.

The COPS package adds a lightweight [OPDS](https://en.wikipedia.org/wiki/OPDS) webserver to your ebook library with search functionality.
